style/ruby/README.md

```diff
@@ -11,6 +11,8 @@ Ruby
 * Avoid explicit return statements.
 * Avoid using semicolons.
 * Avoid bang (!) method names. Prefer descriptive names.
+* Avoid defining classes and modules using shorthand nesting (class A::B)
```

Comment:
Should we flip this wording to match the example?
Something like: `Prefer nested module definitions instead of the shorthand version`